### DISCUSSION TITLE
SCAM

### DISCUSSION THREAD

**Discussion by Rootx10** [community/community#102815] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello, i found scammer on github, he publish product for $40 and after payment he asking for more money, this is the link to project https://github.com/hoglyx/Bitcoin-SenderV2 please be cearfull

**Comment by LuluW8071** [comment]
Looks like he also made a bot accounts (or alts) to star his own repository

**Comment by sachasa** [comment]
Thanks for the info

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Rootx10  👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
